.github/workflows/main.yml

```diff
@@ -53,8 +53,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-latest, windows-latest]
-        node: [14, 16, 17, 18, 20]
+        os: [ubuntu-20.04, macos-12, macos-14, windows-latest]
+        node: [16, 17, 18, 20]
```

Comment:
Node `14` is not available on this new image. As it's pretty old now (node `20` is the LTS), I removed it.

`macos-12` stay to allow for `x64` architecture prebuild (`macos-latest` will move to `arm64` in the future, so set the version explicitely)